### ISSUE TITLE
Bug - [Dual List Selector#basic-with-search] - [Second option is selected instead of first from Available option list]

### ISSUE THREAD

**Issue by rnargotr1710** [patternfly/patternfly-react#12130]
**Describe the problem**
When the focus is on Available Options, the on pressing arrow key, second option in list is selected instead of first:The issue is with Component>Dual List Selector>. And this issue is in Patternfly 5. 

**How do you reproduce the problem?**
1. Reach to the Menus Component>Dual List Selector (https://v5-archive.patternfly.org/components/dual-list-selector/#basic-with-search )
2. Start the voiceover and try to read the available options
3. When the focus is on Available Options, the on pressing arrow key, second option in list is selected instead of first

**Expected behavior**
When the focus is on Available Options, the on pressing arrow key, first option should be selected

**Screenshots**

<img width="1512" height="982" alt="Image" src="https://github.com/user-attachments/assets/212f893b-8521-4ffe-9019-020a8d1ad293" />

**What is your environment?**
 -iOS
